com.unity.netcode.gameobjects/Tests/Editor/com.unity.netcode.editortests.asmdef

```diff
@@ -28,12 +27,5 @@
         "UNITY_INCLUDE_TESTS"
     ],
     "versionDefines": [],
-    "noEngineReferences": false,
-    "versionDefines": [
-        {
-            "name": "com.unity.multiplayer.tools",
-            "expression": "",
-            "define": "MULTIPLAYER_TOOLS"
-        }
-    ]
```

Comment:
Why did you remove this? Did you test NGO tests without tools package installed? The purpose of this is to make sure that NGO tests still compile and can be run by users who don't have tools package installed.

Reply:
Let me look into this...

Reply:
Yeah, that was definitely a good catch.  I think I was comparing it to the testproject editor tests assembly definition.
Either case it is fixed.